test/scripts/basic-test.js

```diff
@@ -238,12 +248,27 @@ describe('<ll-property-images>', function() {
       nodeListAsArray: nodeListAsArray
     };
 
-    var currentDefaultImage = undefined;
+    delete imageData.detail[0].imgId;
+
+    //before running function that switches values
+    expect(imageData.detail[0].isDefault).to.equal(true);  //old default image
+    expect(imageData.detail[1].isDefault).to.equal(false);   //new default image
+
+    // if it's the first time a user has uploaded photos then imageData.detail[0].imgId (passed in from the manager
+    // that's updating the back end, will be undefined. so, we must set imgId of old default to the imgId of the
+    // first index in the nodeList. (We know this is the old default image, bc the first image added is automatically
+    // set to the default image
+    if (imageData.detail[0].imgId === undefined && imagesToUpdate.nodeListAsArray.length > 1) {
+      imagesToUpdate.oldDefaultId = nodeListAsArray[0].imgId;
+    }
```

Comment:
I'm going to be honest. I'm not sure at all what line 251-263 is doing that's relevant. From what I can tell it's just deleting the id and then setting the id back on.